topics/EXP-output.rst

```diff
@@ -54,6 +58,11 @@ the Doxygen generated man pages or the source code itself.
   for any component but makes the most sense for near-equilibrium
   galaxy components such as disks, bulges and halos.
 
+- `orbtrace` will output a select number of orbits to the same file, appending phase space
+   information to the end at each time step. This is helpful for quick looks, but 
+   is not meant to be science product output (i.e. one should probably not try to set 
+   `orbtrace` to the number of disk particles).
```

Comment:
One could run a simulation, select particular particles of interest--e.g. those that passed through some resonance--and rerun or playback to get their orbits.  I suppose with playback, one could run `exp` with only the orbits of interest to accomplish the same thing so maybe you're right.  I completely agree about not setting `orbtrace` to the number of disk particles,  That would be daft.

Reply:
How would you specify /which/ particles should get saved out in the `orbtrace`, if you wanted to re-run the full simulation? 
re:playback, that is a good point to add here with a link to the relevant page

Reply:
`orbtrace` keeps a list of particle indices to dump.  The particle index is assigned sequentially when the phase space is created from the ascii input file and is unique thereafter.

One can use `orbtrace` in two ways:
1. Specify `nbeg`, `norb`, `nskip` to make a list of `norb` particle indices that begin with `nbeg` and increment by `nskip`.
2. Provide a file using the `orbitlist` parameter, a filename containing a list of particles indices to trace.

`orbtrace` predates playback.  In some sense, the `orbitlist` feature has been partly superceded by playback for situations where one might want to rerun the simulation to acquire specific orbits based on criterion from analysis of the already run simulation.  On the other hand, if one had a particle set of tracers of interest, one could assign them zero mass and front load the particle file with these desired tracers before any simulation was run.  Or, one could find the particular orbits in the particle file that one wanted to trace according to some criterion (e.g. all particles with energy between [E_min, E_max]).